No such locale sr_LATN.

Follow on to https://github.com/mozilla-services/puppet-config/pull/250, in the languages list, it includes `sr_LATN`, which results in the below. (There does exist fxa-content-server/app/i18n/sr_Latn, but case matters(?)).

Bad locale=[sr_LATN] missing .key-value-json files in [/data/fxa-content-server/app/i18n/sr_LATN/messages.json]. See locale/README (Error: Cannot find module '/data/fxa-content-server/app/i18n/sr_LATN/messages.json')

--- a/.travis.yml
+++ b/.travis.yml
@@ -22,7 +22,7 @@ install:
   - cd ../..
 # copy over the configuration that can be used to start the server.
   - cp server/config/local.json-dist server/config/local.json
-  - grunt po2json
+  - grunt l10n-create-json
   - npm start &
 # now run the tests!
 script: grunt lint && npm run-script test-server && npm run-script test-remote

--- a/grunttasks/po2json.js
+++ b/grunttasks/po2json.js
@@ -6,6 +6,7 @@
 // .po files are expected to already be downloaded.
 
 const path = require('path');
+const i18n = require('i18n-abide');
 
 module.exports = function (grunt) {
   'use strict';
@@ -32,7 +33,10 @@ module.exports = function (grunt) {
          * content-server.po should have an output filename of messages.json
          */
         var matches = /^locale\/([^\/]+)\/LC_MESSAGES\/(.*)$/.exec(file);
-        var locale = matches[1].replace(/-/g, '_');
+
+        // In order to make sure the locale is in a form that i18n-abide expects
+        // we convert it from locale to langauge back to locale.
+        var locale = i18n.localeFrom(i18n.languageFrom(matches[1]));
         var filename = matches[2];
         if (filename === 'server.po') {
           filename = 'messages.json';

